package.json

```diff
@@ -47,13 +47,10 @@
         "eslint": "eslint .",
         "ejs-lint": "mocha test/ejslint.js",
         "ejslint": "ejslint",
-        "pretest": "npm run lint",
```

Comment:
is it launch somewhere ? Or do we lost the lint ?

Reply:
My bad, I forgot to add it back